deployment/ccip/changeset/v1_6/cs_update_rmn_config.go

```diff
@@ -52,6 +60,14 @@ func (c SetRMNRemoteOnRMNProxyConfig) Validate(state changeset.CCIPOnChainState)
 		if chainState.RMNProxy == nil {
 			return fmt.Errorf("RMNProxy not found for chain %d", chain)
 		}
+		chain := e.Chains[chain]
+
+		if err := commoncs.ValidateOwnership(e.GetContext(), c.MCMSConfig != nil, chain.DeployerKey.From, chainState.Timelock.Address(), chainState.RMNRemote); err != nil {
```

Comment:
Do we need RMNRemote to be owned by Timelock at this point? probably safer though.

Reply:
It would ensure that only MCMS could change the config on the RMNRemote, which is nice. It also provides redundant curse / uncurse protection